packages/discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -446,9 +446,8 @@ def delete_track(params: ManageEntityParameters):
         params.block_datetime,
     )
     deleted_track.is_delete = True
+    # Detach this track from the parent if it is a stem
     deleted_track.stem_of = null()
-    deleted_track.remix_of = null()
```

Comment:
think i'm missing context here, why do we need to null these fields out (or not) when a track is deleted? If it's to remove a relationship, wouldn't we want to null out `remix_of` too?

Reply:
it used to do this, and didn't do a particularly good job because it left an artifact in the remixes table.
I removed these because it definitely causes problems with how we should treat something after deleted, e.g. "should we show a notification for this track?"